modules/ROOT/pages/Development/BeginnersGuide/project_setup.adoc

```diff
@@ -110,9 +124,15 @@ Make sure you run the above command from Command Prompt and not Powershell. It i
 You'll want to compile the project now to ensure it generates correctly.
 
 Do this by opening the .sln file, make sure you have in the toolbar `Development Editor` and `Win64` selected.
-On the right side in solution explorer just r-click the `FactoryGame` project and hit `Build`. This will take some time; go pet some lizard doggos as you wait.
+On the right side in Solution Explorer just right click on
+the `FactoryGame` project and hit `Build`.
+This will take some time; go pet some lizard doggos as you wait.
+You'll know it's done when the little box-with-cubes-piling-into-it
+icon in the blue bar at the bottom of Visual Studio goes away.
+You can monitor its progress from the Output log if desired.
 
-This creates the binaries used by the Editor, allowing it to interact with the native SML code or native code written by you.
+This creates the binaries used by the Editor,
+allowing it to interact with the native SML code or native code written by you.
 
 When you want to test the mod, make sure to build the project for `Shipping`.
```

Comment:
We should put more emphasis on the fact it is needed to build for Shipping after creating a plugin (maybe even before, to verify)